pipeline-adapters/mlrun-pipelines-kfp-v2/pyproject.toml

```diff
@@ -1,9 +1,9 @@
 [project]
 name = "mlrun-pipelines-kfp-v2"
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
```suggestion
version = "0.6.0"
```

Reply:
the change you are doing here is not a "patch" modification but rather a minor since we now dropping backwards compatibility support of python 3.9 - meaning, at least updating minor here.